.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: false
+
+language: go
+
+go:
+  - 1.4
+  - 1.5
+  - 1.6
+  - 1.7
+  - 1.8
+  - tip
+
+before_install:
+  - go get github.com/docker/docker/api/types/reference
```

Comment:
not sure if that is enough actually, as i have not tested it on a fresh machine yet. too bad that travis doesn't run already for that PR